tldr/cross_file_calls/builder.py

```
@@ -0,0 +1,192 @@
+"""
+Call graph builder for cross-file call analysis.
+"""
+
+import os
+import time
+from typing import Dict, List, Optional, Set
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)